addons/website/static/src/js/website.snippets.editor.js

```diff
@@ -1308,6 +1308,107 @@
             this.$target.find('.item.text_image, .item.image_text, .item.text_only').find('.container > .carousel-caption > div, .container > img.carousel-image').attr('contentEditable', 'true');
         },
     });
+
+    website.editor.countdown = website.editor.Dialog.extend({
+        template: 'website.countdown_configuration',
+        events : _.extend({}, website.editor.Dialog.prototype.events, {
+            'click button[data-action=set_countdown]': 'set_countdown',
+        }),
+        init: function (parent) {
+            this.$target = parent.$target;
+            return this._super();
+        },
+        start: function () {
+            var self = this;
+            this.date_input = self.$el.find("#web_countdown_date").datepicker({
+                minDate: 0,
+            });
+            return this._super();
+        },
+        set_countdown: function () {
+            var self = this;
+            var release_date = this.date_input.datepicker( 'getDate' );
+            var hour = document.getElementById("web_countdown_hour").value;
+            var minute = document.getElementById("web_countdown_min").value;
+            var second = document.getElementById("web_countdown_sec").value;
+            var local_date  = new Date(release_date.getFullYear(),release_date.getMonth(),release_date.getDate(), hour, minute, second);
+            var add_warning = function (msg){
+                self.$el.find(".alert").remove();
+                self.$el.find(".modal-body").append('<div class="alert alert-danger mt8">'+ msg +'</div>');
+            }
+            if(hour > 23 | hour < 0 ){
+                add_warning("Enter valid HOURS (Between 0 to 23)");
+            }
+            else if(minute > 59 | minute < 0 ){
+                add_warning("Enter valid MINUTES (Between 0 to 59)");
+            }
+            else if(second > 59 | second < 0 ){
+                add_warning("Enter valid SECONDS (Between 0 to 59)");
+            }
```

Comment:
As per above code of if...else, If I added a date and time like, 123 hour 345 minute and 234 seconds, now when I try to save it will give only one alert message that is "Enter valid HOURS (Between 0 to 23)"

Don't you think so it should show all three messages as I entered all three inputs wrong